src_py/sprite.py

```diff
@@ -215,7 +220,20 @@ def alive(self):
         return truth(self.__g)
 
     def __repr__(self):
-        return "<%s sprite(in %d groups)>" % (self.__class__.__name__, len(self.__g))
+        return "<%s sprite(in %d groups)>" % (self.__class__.__name__,
```

Comment:
Lowercase sprite here was an old mistake. 
- Changing it now might break some doctest based tests, but that's about it.
- Fixing it would make it slightly more readable.

So, probably worth fixing to `<Sprite...>`.

Reply:
👍